Tests/JWTTests/JWTTests.swift

```diff
@@ -124,17 +124,40 @@ class JWTKitTests: XCTestCase {
         let app = Application(.testing)
         defer { app.shutdown() }
 
-        let apple = JWKSCache(keyURL: "https://appleid.apple.com/auth/keys", on: app)
+        app.client.configuration.ignoreUncleanSSLShutdown = true
 
-        let request = Request(application: app, on: app.eventLoopGroup.next())
-        let keys = try apple.keys(on: request).wait()
+        let apple = JWKSCache(
+            keyURL: "https://www.googleapis.com/oauth2/v3/certs",
+            client: app.client
+        )
 
-        let key = keys.find(identifier: "AIDOPK1", type: .rsa)
-        XCTAssertNotNil(key)
-        XCTAssertNotNil(key!.algorithm == .rs256)
+        app.get("keys") { req in
+            apple.keys(on: req).map { jwks in
+                jwks.keys.count
+            }
+        }
+
+        try app.test(.GET, "keys") { res in
```

Comment:
This is just asking if you got keys.  Is that really OK?  Shouldn't we be looking for something specific, like I did with the apple one?

Reply:
To get the count that means the `JWKS` were decoded successfully so I think that's sufficient for this test.